app/docker/components/datatables/networks-datatable/networksDatatableController.js

```diff
@@ -0,0 +1,22 @@
+angular.module('portainer.docker')
+  .controller('NetworksDatatableController', ['$scope', '$controller',
+    function ($scope, $controller) {
+      angular.extend(this, $controller('GenericDatatableController', {$scope: $scope}));
+
+      var PREDEFINED_NETWORKS = ['host', 'bridge', 'none'];
```

Comment:
```suggestion
      var predefinedNetworks = ['host', 'bridge', 'none'];
```

Let's use regular variable declaration here to keep the codebase consistent.

Reply:
Done